### PR TITLE
Orthocode update default keymap, and fix encoder function for keymaps built in configurator

### PR DESCRIPTION
**Pull request by Jrodna**
## Description

Updating the default keymap to match the one shipped on the boards.
Moving the encoder function definition to orthocode.c so it works by default for those using the online qmk configurator

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/orthocode/orthocode.c [hunk 1]
```diff
@@ -13,4 +13,16 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include "orthocode.h"
\ No newline at end of file
+#include "orthocode.h"
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    // Volume control
```
Comment (drashna) [inline keyboards/orthocode/orthocode.c:20]:
```suggestion
    if (!encoder_update_user(index, clockwise)) { return false; }
    // Volume control
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/orthocode/orthocode.c (hunk 1 above)
Comment (Jrodna) [reply keyboards/orthocode/orthocode.c:21]: Added this in fix commit

**Review by drashna — APPROVED** [review]

**Comment by Jrodna** [comment]
Is there anyway to ping someone to get this merged? Boards are starting to arrive with customers so I want the online configurator to be working properly.

**Comment by drashna** [comment]
> Is there anyway to ping someone to get this merged? Boards are starting to arrive with customers so I want the online configurator to be working properly.

Mostly, that's what the point of requesting the "collaborators" group is for. 

https://docs.qmk.fm/#/pr_checklist?id=review-process
> Additionally, PR reviews are something that is done in our free time. We are not paid nor compensated for the time we spend reviewing, as it is a labor of love. As such, this means that it can take time for us to get to your Pull Request. Things like family, or life can get in the way of us getting to PRs, and burnout is a serious concern. The QMK firmware repository averages 200 PRs opened and 200 PRs merged every month, so please have patience.

**Comment by Jrodna** [comment]
> > Is there anyway to ping someone to get this merged? Boards are starting to arrive with customers so I want the online configurator to be working properly.
> 
> Mostly, that's what the point of requesting the "collaborators" group is for.
> 
> https://docs.qmk.fm/#/pr_checklist?id=review-process
> 
> > Additionally, PR reviews are something that is done in our free time. We are not paid nor compensated for the time we spend reviewing, as it is a labor of love. As such, this means that it can take time for us to get to your Pull Request. Things like family, or life can get in the way of us getting to PRs, and burnout is a serious concern. The QMK firmware repository averages 200 PRs opened and 200 PRs merged every month, so please have patience.

If that's the case how can I help, do I need to be set as a collaberator before there is any point reviewing? Or can I just start looking at PRs and helping out?

**Comment by tzarc** [comment]
Anyone can review PRs; the difference is that collaborators are the direct maintainers of QMK itself and is not a group that can be joined at will. Only collaborators show up as a green tick as an approving review, but reviews from others do help, especially people confirming that they've tested the PR locally.

**Review by tzarc — APPROVED** [review]

**Pull request merged**